### ISSUE TITLE
Deploy RDS Instance with Tables

### ISSUE THREAD

**Issue by ciaransweet** [NASA-IMPACT/hls-sentinel2-downloader-serverless#2]
# TL;DR

Currently the downloader is setup as a MySQL database in RDS, these need to be created to be used with this project.

The tables it includes are:
* granule_count
* granules
* status

It is currently a `db.m5.large`

![image](https://user-images.githubusercontent.com/9111975/102629423-0c4e3b80-4143-11eb-94ad-7318adadea83.png)
(And status of course)

# Info
CREATE Statements for each table are available below (Retrieved via `SHOW CREATE TABLE <tablename>`):

**granule_count**

```sql
CREATE TABLE `granule_count` (
  `date` date NOT NULL,
  `available_links` int(11) NOT NULL,
  `fetched_links` int(11) NOT NULL,
  `last_fetched_time` datetime NOT NULL,
  PRIMARY KEY (`date`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1
```

**granule**

```sql
CREATE TABLE `granule` (
  `id` varchar(255) NOT NULL,
  `filename` text NOT NULL,
  `tileid` text NOT NULL,
  `size` bigint(20) NOT NULL,
  `checksum` text NOT NULL,
  `beginposition` datetime NOT NULL,
  `endposition` datetime NOT NULL,
  `ingestiondate` datetime NOT NULL,
  `download_url` text NOT NULL,
  `downloaded` tinyint(1) NOT NULL,
  `in_progress` tinyint(1) NOT NULL,
  `uploaded` tinyint(1) NOT NULL,
  `ignore_file` tinyint(1) NOT NULL,
  `retry` int(11) NOT NULL,
  `download_started` datetime DEFAULT NULL,
  `download_finished` datetime DEFAULT NULL,
  `download_failed` tinyint(1) NOT NULL DEFAULT '0',
  `expired` tinyint(1) NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1
```

**status**

```sql
CREATE TABLE `status` (
  `key_name` varchar(255) NOT NULL,
  `value` text NOT NULL,
  PRIMARY KEY (`key_name`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1
```

# Acceptance criteria

* RDS Instance deployed via the CDK stack
* The user is automatically created (see @alukach's DB Bootstrap)
* The credentials are stored in Secrets Manager
